client/src/plugins/version-info/ReleaseInfo.js

```diff
@@ -44,16 +44,16 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Enhanced BPMN ad-hoc sub-process support</h4>
-          You can now configure new ad-hoc sub-process properties such as output collection and implementation as a job worker.
+          <h4>Support for Camunda 8.8 and 7.24</h4>
+          Camunda 8.8 and 7.24 are now fully supported.
         </li>
         <li>
-          <h4>Support for deployment using the new Orchestration Cluster REST API</h4>
-          The Modeler now supports deploying through the new <a href="https://docs.camunda.io/docs/8.8/apis-tools/orchestration-cluster-api-rest/orchestration-cluster-api-rest-overview/">Orchestration Cluster REST API</a>.
+          <h4>Task testing</h4>
```

Comment:
```suggestion
          <h4>Introducing task testing</h4>
```

We could be a little bit more engaging here.